From 69beb37db5e04785da6b5e9c9739a74820137d27 Mon Sep 17 00:00:00 2001
From: Tian Yang <tiany@codeaurora.org>
Date: Tue, 15 Nov 2016 09:09:11 -0600
Subject: [PATCH] Remove pfmemalloc initialization

We now reject recycling pfmemalloc SKBs, so carrying this flag over is
no longer necessary.

Change-Id: I5ca952d2f555280c5f610f54b4ec697a16a939d9
Signed-off-by: Cristian Prundeanu <cprundea@codeaurora.org>
Signed-off-by: Casey Chen <kexinc@codeaurora.org>
---
 net/core/skbuff_recycle.c | 2 --
 1 file changed, 2 deletions(-)

--- a/net/core/skbuff_recycle.c
+++ b/net/core/skbuff_recycle.c
@@ -99,7 +99,6 @@ inline struct sk_buff *skb_recycler_allo
 
 	if (likely(skb)) {
 		struct skb_shared_info *shinfo;
-		bool pfmemalloc = skb->pfmemalloc;
 
 		/* We're about to write a large amount to the skb to
 		 * zero most of the structure so prefetch the start
@@ -111,7 +110,6 @@ inline struct sk_buff *skb_recycler_allo
 
 		zero_struct(skb, offsetof(struct sk_buff, tail));
 		refcount_set(&skb->users, 1);
-		skb->pfmemalloc = pfmemalloc;
 		skb->mac_header = (typeof(skb->mac_header))~0U;
 		skb->transport_header = (typeof(skb->transport_header))~0U;
 		zero_struct(shinfo, offsetof(struct skb_shared_info, dataref));
